src/App.jsx

```diff
@@ -0,0 +1,234 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import './App.css';
+import { Goods } from './components';
+
+const goodsFromServer = [
+  'Dumplings',
+  'Carrot',
+  'Eggs',
+  'Ice cream',
+  'Apple',
+  'Bread',
+  'Fish',
+  'Honey',
+  'Jam',
+  'Garlic',
+];
+
+const defaultLengthlimit = 1;
+
+function Start({ runAplication }) {
+  return (
+    <button
+      type="button"
+      className="start-button"
+      onClick={runAplication}
+    >
+      Start
+    </button>
+  );
+}
+
+function Stop({ runAplication }) {
```

Comment:
```suggestion
function stopApp({ runAplication }) {
```
start/stop слишком обобщенное название, уточняй что именно ты стартуешь/останавливаешь